cucumber-expressions/java/pom.xml

```diff
@@ -46,11 +47,20 @@
             <version>2.8.5</version>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest-core</artifactId>
+            <version>${hamcrest.version}</version>
```

Comment:
Could you change this to use a literal value rather than a variable? The `update-dependencies` make target only works when the versions are literal:

https://github.com/cucumber/cucumber/blob/0e63c88e101b29da3f69d95b395197a486d25963/cucumber-expressions/java/default.mk#L14

Reply:
Sure.

Though I'm not a fan of automatically updating dependencies. Maven does not understand the concept of pre-releases so will happily jump from 1.0.0 to 2.0.0-M1. This is usually not desirable as these releases can be quite buggy.

Reply:
> The update-dependencies make target only works when the versions are literal

You can use `versions:update-properties` for this:

http://www.mojohaus.org/versions-maven-plugin/update-properties-mojo.html